cluster/examples/kubernetes/ceph/common.yaml

```diff
@@ -290,6 +290,7 @@ rules:
       - configmaps
       - nodes
       - nodes/proxy
+      - persistentvolumes
```

Comment:
Whatever changes we make in common.yaml, we also need to make the same changes for the helm chart manifests. See [clusterrole.yaml](https://github.com/rook/rook/blob/master/cluster/charts/rook-ceph/templates/clusterrole.yaml)